.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,42 @@
+name: 🐛 Bug Report
+description: Do you find a bug? Create a report to help us improve.
+title: "[BUG]:"
+labels: ["bug", "blocked"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a bug! Please fill out the details below to help us understand and resolve it.
+  
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Provide a clear and concise description of the bug. Include steps to reproduce the behavior, the expected result, and the actual result.
```

```suggestion
      description: Provide a clear and concise description of the bug. Include steps to reproduce the behavior, the expected result, and the actual result.
```

Are the steps to reproduce appropriate here? Would it be better to be 'what I am reading' and 'what I think this should say' or something that's less software development and more contextually appropriate?